src/main/java/com/oltpbenchmark/benchmarks/templated/util/TemplatedValue.java

```diff
@@ -0,0 +1,112 @@
+package com.oltpbenchmark.benchmarks.templated.util;
+
+import java.util.Random;
+
+import com.oltpbenchmark.distributions.ScrambledZipfianGenerator;
+import com.oltpbenchmark.distributions.ZipfianGenerator;
+
+/**
+ * This class is used to store information about the values
+ * used in templated benchmarks. It can hold static values but also
+ * generators for value distributions
+ */
+public class TemplatedValue {
+    ValueGenerator distribution;
+    Long min;
+    Long max;
+    Long seed;
+    String value;
+    String minS;
+    String maxS;
+
+    public final Object generatorObject;
+
+    /**
+     * 
+     * @param dist  The desired value distribution
+     * @param min   Minimum value. Default is 0
+     * @param max   Maximum value. Default is 1
+     * @param seed  The seed for the random generator. Default is 0
+     * @param value Value that is used if no distribution is given
```

Comment:
If `value` is the more common expectation, I might suggest to provide that as the first element.

Another suggestion would be to create to constructors that call into this (private) one:
- One with distribution related content.
- The other with just a value.